.rubocop.yml

```diff
@@ -14,7 +14,7 @@ SingleLineBlockParams:
 # resolutions and Github's pull request view. But let's open the threshold a
 # little bit
 Metrics/LineLength:
-  Max: 105
+  Max: 110
```

Comment:
To be honest, if changing the offending code is not possible I would prefer to disable the check just for the affected piece of code instead of opening the door for the whole repository.